sympy/functions/special/gamma_functions.py

```diff
@@ -1009,3 +1009,67 @@ def trigamma(x):
     .. [3] http://functions.wolfram.com/GammaBetaErf/PolyGamma2/
     """
     return polygamma(1, x)
+
+def multivariate_gamma(x, p):
+    r"""
+    The multivariate gamma function is a generalization of the gamma function i.e,
+
+    .. math::
+        \Gamma_p(z) = \pi^{p(p-1)/4}\prod_{k=1}^p \Gamma[z + (1 - k)/2].
+
+    Special case, multivariate_gamma(x, 1) = gamma(x)
+
+    Parameter
```

Comment:
This should be named `Parameters` and a new line should be added before `p: ...`.

Reply:
Done.